### PR TITLE
Update/035

### PR DESCRIPTION
**Pull request by elvece**

### CODE REVIEW

File: site/source/conf.py [hunk 1]
```diff
@@ -70,14 +70,15 @@
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ["0.2.x", "0.3.1.x", "0.3.2.x", "master"]
+BRANCHES = ["0.2.x", "0.3.1.x", "0.3.2.x", "0.3.4.x"]
```
Comment (MattDHill) [inline site/source/conf.py:73]: I think this needs 0.3.5.x

File: site/source/getting-started/initial-setup.rst [hunk 2]
```diff
@@ -37,17 +37,9 @@ Follow the instructions below to set up your server for the first time.
 		:width: 50%
 		:alt: SSD Initialization
 
-#. Click the ``Download`` button to download a file containing the URLs (`.local` and `.onion`) and Root CA of your server. Keep this file for future reference.
```
Comment (MattDHill) [inline site/source/getting-started/initial-setup.rst:40]: Why did you delete this? It has not changed from 0.3.4.x. Users should continue to download their permanent address info.

Comment (elvece) [reply site/source/getting-started/initial-setup.rst:40]: Because I was verifying in kisok mode and forgot that this has a different view. Will add back in.

File: site/source/conf.py (hunk 1 above)
Comment (elvece) [reply site/source/conf.py:73]: I opted to rename the latest version to 0.3.5.x, so we can continue pushing to master and deploying from there if any changes are needed while still in the 0.3.5.x series. I think this is easier to maintain than having an additional 0.3.5.x branch. The flow would be to cut a branch when we move on to the next series (eg. for 0.3.4.x).

**Comment by gStart9** [comment]
Should all these Pro references change to Pure?

**Comment by MattDHill** [comment]
Yes, but there really shouldn't be much references to our hardware in the docs

File: site/source/conf.py (hunk 1 above)
Comment (MattDHill) [reply site/source/conf.py:73]: I agree. I'm wondering why 0.3.5.x is not in the BRANCHES array

File: site/source/conf.py [hunk 3]
```diff
@@ -70,14 +70,15 @@
...
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Defines which version is considered to be the latest stable version.
 # Must be listed in smv_tag_whitelist or smv_branch_whitelist.
 smv_latest_version = "master"
-smv_rename_latest_version = "latest"
+smv_rename_latest_version = "0.3.5.x"
```
Comment (MattDHill) [inline site/source/conf.py:78]: Also, this should probably be 0.3.4.4 in order to maximize backwards compatibility

File: site/source/getting-started/initial-setup.rst (hunk 2 above)
Comment (elvece) [reply site/source/getting-started/initial-setup.rst]: Restored

**Review by kn0wmad — APPROVED** [review]
